civictechprojects/static/css/partials/_LandingController.scss

```diff
@@ -42,10 +42,16 @@
   margin: 5px; /* override this on desktop */
 }
 .LandingController-root .SplashScreen-content {
-  margin: auto;
+  padding-bottom: 2rem;
+  margin-left: auto;
+  margin-right: auto;
+}
+.LandingController-root .SplashScreen-content h1 {
+  font-size: 3rem;
 }
 .LandingController-topsplash .SplashScreen-content h1 {
   font-size: 4rem;
```

Comment:
In case I'm unavailable to finish this item: For mobile text changes still under discussion, change them here, but first copy and paste these font-size rules as-is into the desktop media query starting on line 206.